src/app/Instance/BridgeOverview/Components/BOTableView.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import SEStatusLabel from "@app/components/SEStatusLabel/SEStatusLabel";
+import {
+  Button,
+  Divider,
+  Flex,
+  FlexItem,
+  Label,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  Truncate,
+} from "@patternfly/react-core";
+import {
+  ActionsColumn,
+  TableComposable,
+  Tbody,
+  Td,
+  Tr,
+} from "@patternfly/react-table";
+import { ManagedResourceStatus } from "@rhoas/smart-events-management-sdk";
+import { DemoData } from "../BridgeOverview";
+
+interface BOTableViewProps {
+  demoData: DemoData[];
+  name: string;
+}
+
+export const BOTableView = (props: BOTableViewProps): JSX.Element => {
+  const rowActions = [{ title: "Edit" }, { title: "Delete" }];
+
+  const { demoData, name } = props;
+
+  return (
+    <>
+      <Split hasGutter>
+        <SplitItem isFilled>
+          <TextContent>
+            <Text component="h4">{name}</Text>
+          </TextContent>
+        </SplitItem>
+        <SplitItem>
+          <Button variant="secondary">Create {name}</Button>
+        </SplitItem>
+      </Split>
+      <Divider style={{ paddingTop: "1rem" }} />
+      <TableComposable variant="compact">
+        {demoData &&
+          demoData.map((processor, rowIndex) => (
+            <Tbody key={rowIndex} style={{ border: "0rem" }}>
```

Comment:
Why this `border` rule? It doesn't seem to have any effect.

Reply:
Yeah, but at the lower resolution it has some effect. So I thought of adding inline style. I will move this into seperate CSS file.